src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # BinSkim Release History
 
+## Unreleased
+
+* BUGFIX: Fix ` KeyNotFoundException` raised on `BA2006.BuildWithSecureTools` when required properties are removed from the config file. [#565](https://github.com/microsoft/binskim/pull/565)
```

Comment:
>KeyNotFoundException [](http://example.com/codeflow?start=16&length=20)

There is an unnecessary space before keynotfoundexception. it should be 'raised by' not raised on. We should mention the configuration option where this occurs as it is data a customer is likely to search on (please always try to imagine your way into this).

I will make these changes and update the branch right now.